docs/rfc/active/000-ml_models/README.md

```diff
@@ -0,0 +1,121 @@
+  - Relevant groups that were not represented in the evaluation dataset
+  - Recommendations for model use
+  - Ideal characteristics of an evaluation dataset
+- Institutional Memory: Institutional knowledge for easy search and discovery.
+- Status: Captures if the model has been soft deleted or not.
+- Cost: Cost associated with the model based on the project/component this model belongs to.
+- Deprecation: Captures if the model has been deprecated or not.
+
+### ML Feature metadata
+- Feature Properties: Basic information about the ML Feature
+  - Description of the feature
+  - Data type of the feature i.e. boolean, text, etc
+- Ownership: Owners of the ML Feature.
+- Institutional Memory: Institutional knowledge for easy search and discovery.
+- Status: Captures if the feature has been soft deleted or not.
+- Deprecation: Captures if the feature has been deprecated or not.
+
+### Metadata graph
+![ml_model_graph](ml_model_graph.png)
+
+An example metadata graph with complete data lineage picture is shown above. Below are the main edges of the graph
+1. Evaluation dataset contains data used for quantitative analyses hence ML Model is connected to the evaluation dataset(s) through `DownstreamOf` edge
+2. Training dataset(s) contain the training data hence ML Model is connected to the training dataset(s) through `DownstreamOf` edge.
+3. ML Model is connected to `DataProcess` entity which captures the training execution through a (newly proposed) `TrainedBy` edge.
+4. `DataProcess` entity itself uses the training dataset(s) (mentioned in 2) as it's input and hence is connected to the training datasets through `Consumes` edge.
+5. ML Model is connected to ML Feature(s) through `Contains` edge.
+6. Results of the performance of ML Model can be viewed in a dashboard and is therefore connected to `Dashboard` entity through `Produces` edge.
+
+## How we teach this
+
+We should create/update user guides to educate users for:
+ - Search & discovery experience (how to find a machine learning model in DataHub)
+ - Lineage experience (how to find different entities connected to the machine learning model)
+
+## Alternatives
+A machine learning model could as well store a model ID which uniquely identifies a machine learning model in the machine learning model lifecycle management system. This can then be the only component of `MLModelUrn`. However we chose the approach of modeling `MLModelUrn` similar to `DatasetUrn`.
+
+## Rollout / Adoption Strategy
+
+The design is supposed to be generic enough that any user of DataHub should easily be able
+to onboard their ML model and ML feature metadata to DataHub irrespective of their machine learning platform.
+
+Only thing users will need to do is to write an ETL script customized for their machine learning platform (if it's not already provided in DataHub repo). This ETL script will construct and emit ML model and ML feature metadata in the form of [MCEs](../../../what/MXE.md).
+
+## Unresolved questions
```

Comment:
Call it Future work instead of Unresolved questions.

Reply:
Done